home/gettingstarted/0.7/persistors.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docsdefault07
+title: Persistors
+permalink: /home/gettingstarted/0.7/persistors/index.html
+
+smversion: 0.7
+partof: getting-started
+num: 6
+outof: 50
+---
+
+
+`Persistor`s are used to store results from executed benchmarks.
+
+## Persistor.None
+
+It actually does no persisting at all.
+
+## SerializationPersistor
+
+It uses `Java serialization` under the hood.  
+
+Note that this persistor is not a good choice if you want to keep you results while upgrading `ScalaMeter` version.
+
+## JSONSerializationPersistor
+
+It writes a regression data to disk as JSON. 
+
+It's the fastest among `Persistor`s, although it produces the largest output files.   
+
+## GZIPJSONSerializationPersistor
+
+Default choice for all `HTMLReport` descendants. 
+
+It works like `JSONSerializationPersistor`, but instead of storing a plain JSON on disk it gzips it firsts.
+This allows to be much more space efficient than `JSONSerializationPersistor` 
+whilst being a little slower (but still faster than `Serialization Persistor`).
+
+## Pickler
+
+For JSON-based persistors `Parameters` are stored as Base64-encoded strings. 
+This enforces that every `Parameter` (and thus `Gen` implementation) requires `Pickler`. 
```

Comment:
You can add a sentence in-between: "These Base64-encoded strings can be produced by using `Pickler` classes.", then instead of "This enforces that..." change to "For this reason,".